build.gradle

```diff
@@ -26,7 +26,7 @@ task clean(type: Delete) {
 ext {
     bintrayUserProperty = ""
     bintrayApiKeyProperty = ""
-    libraryVersion = '1.0.2'
+    libraryVersion = '1.0.3'
```

Comment:
Let us create branch `release/1.0.3` and update here.

Reply:
OK. I'll revert the change.